content/cloud-docs/users-teams-organizations/single-sign-on/azure-ad.mdx

```diff
@@ -47,25 +44,29 @@ For more information on the listed features, visit the [Microsoft Azure AD SAML
 
 1. Save, and you should see a completed Terraform Cloud SAML configuration.
 
-1. Copy Entity ID and Assertion Consumer Service URL.
+1. Copy Entity ID and Reply URL.
 
 ## Configuration (Microsoft Azure AD)
 
 1. In the Azure portal, on the **Terraform Cloud** application integration page, find the **Manage** section and select **single sign-on**.
 1. On the **Select a single sign-on method** page, select **SAML**.
 1. On the **Set up single sign-on with SAML** page, click the edit/pen icon for **Basic SAML Configuration** to edit the settings.
    1. In the **Identifier** text box, paste the **Entity ID**.
-   1. In the **Reply URL** text box, paste the **Assertion Consumer Service URL**.
+   1. In the **Reply URL** text box, paste the **Reply URL**.
    1. In the **Sign-on URL** text box, type the URL: `https://app.terraform.io/session`
    1. Select **Save**.
+1. On the **Single sign-on** page, download the `Certificate (Base64)` file from under SAML Signing Certificate
 1. In the app's overview page, find the **Manage** section and select **Users and groups**.
 1. Select **Add user**, then select **Users and groups** in the **Add Assignment** dialog.
 1. In the **Users and groups** dialog, select your user from the Users list, then click the **Select** button at the bottom of the screen.
 1. If you are expecting a role to be assigned to the users, you can select it from the **Select a role** dropdown. If no role has been set up for this app, you see "Default Access" role selected.
 1. In the **Add Assignment** dialog, click the **Assign** button.
 
 ## Configuration (Terraform Cloud)
-
+1. Edit settings of your Azure SSO configuration
+   1. Go to **Public Certificate** 
+   1. Paste the contents of the SAML Signing Certificate downloaded from Microsoft Azure AD
+   1. Save Settings  
```

Comment:
Okay a couple quick notes here - I'm thinking this first one "go to" - is that a button? a dropdown? I also think - the last thing should be "Click **Save**". We want to phrase it in terms of an action and reference the exact UI thing.

```suggestion
To edit your Azure SSO configuration settings:

   1. Go to **Public Certificate**. 
   1. Paste the contents of the SAML Signing Certificate you downloaded from Microsoft Azure AD.
   1. Save Settings.  
```